src/main/java/org/prebid/server/auction/requestfactory/AmpRequestFactory.java

```diff
@@ -184,6 +187,41 @@ public AmpRequestFactory(StoredRequestProcessor storedRequestProcessor,
         return Future.succeededFuture(bidRequest);
     }
 
+    private static ConsentParam consentParamFromQueryStringParams(HttpRequestContext httpRequest) {
+        final ConsentType specifiedConsentType = ConsentType.from(httpRequest.getQueryParams().get(CONSENT_TYPE_PARAM));
+        final CaseInsensitiveMultiMap queryParams = httpRequest.getQueryParams();
+
+        final String consentStringFromConsentParam = queryParams.get(CONSENT_PARAM);
+        final String consentStringFromGdprConsentParam = queryParams.get(GDPR_CONSENT_PARAM);
```

Comment:
Pls rename to gdprConsentParam same for consentStringFromConsentParam. Too long naming should be only if another name can not be used